### PR TITLE
4 get articles

### PR DESCRIPTION
**Pull request by deanlicious**

### CODE REVIEW

File: controllers/articles.controllers.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const { fetchArticles } = require("../models/articles.models");
+
+exports.getArticles = (req, res, next) => {
+  const { article_id } = req.params;
+  fetchArticles(article_id)
```
Comment (nc-niamh-smith) [inline controllers/articles.controllers.js:5]: this is a misleading name - do you want to get an article or multiple?

File: models/articles.models.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const db = require("../db/connection");
+
+exports.fetchArticles = (article_id) => {
```
Comment (nc-niamh-smith) [inline models/articles.models.js:3]: same here - surely you just want the one article?

File: __tests__/articles.test.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+
+afterAll(() => db.end());
+
+beforeEach(() => seed(data));
+
+describe("/api", () => {
+  describe("/articles", () => {
```
Comment (nc-niamh-smith) [inline __tests__/articles.test.js:12]: i'm confused - are you on the endpoint of getArticles or getArticleById?

**Review by nc-niamh-smith — COMMENTED** [review]
Little bit confused here Dean - are you looking for one article by id or for all articles? Make sure your naming conventions match what you're looking for :)

**Comment by deanlicious** [comment]
@nc-niamh-smith updated names of function required for endpoint GET /api/articles/:article_id to be more consistent.

see https://github.com/deanlicious/nc-news/pull/7 for update

**Pull request merged**